src/main/kotlin/no/digdir/fdk/searchservice/model/SearchOperation.kt

```diff
@@ -16,5 +16,6 @@ data class SearchFilters(
   val spatial: String?,
   val provenance: String?,
   val los: String?,
-  val orgPath: String?
+  val orgPath: String?,
+  val collection: SearchCollection?
```

Comment:
collection?

Reply:
Det var det det hette i søket på data.norge:
<img width="631" alt="image" src="https://github.com/Informasjonsforvaltning/fdk-search-service/assets/48259307/6e991bb4-b76d-4af9-ae10-661cfbb9f27d">

Reply:
vi får refaktorere det og, bruk feltnavnet `formats` elns

Reply:
Skal jeg fjerne field også? Altså er formats: List<String> ?

Reply:
ja, men i tillegg kan du også fint refaktorere til slik vi gjør det i concept-catalog, som jeg linka til i den andre kommentaren. Da blir det `formats: SearchFilter<List<String>>?`

Reply:
eventuelt så kan du gå for `formats: List<String>?` nå og så refaktorerer vi alle filtrene til åå bruke SearchFilter senere